prow/cluster/jobs/IBM/ibm-licensing-operator/IBM.ibm-licensing-operator.release-4.0.yaml

```diff
@@ -0,0 +1,276 @@
+presubmits:
+  IBM/ibm-licensing-operator:
+  - name: check_ibm-licensing-operator
+    cluster: default
+    branches:
+    - ^release-4.0$
+    decorate: true
+    always_run: true
+    path_alias: github.com/IBM/ibm-licensing-operator
+    rerun_command: /test check ibm-licensing-operator
+    spec:
+      containers:
+      - command:
+        - make
+        - check
+        image: quay.io/multicloudlab/check-tool:v20200817-b1f2b4c05
+        name: ""
+        securityContext:
+          privileged: true
+    trigger: '(?m)^/test (?:.*? )?check(?: .*?)?$'
+  - name: test_ibm-licensing-operator-amd64
+    cluster: default
+    branches:
+    - ^master$
```

Comment:
@SzymonKowalczyk All the `- ^master$` suppose to be replaced with `- ^release-4.0$` in this file

Reply:
Thank tor notice - corrected